clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2922,18 +2922,21 @@ static void emitUsed(CodeGenModule &CGM, StringRef Name,
   if (List.empty())
     return;
 
+  // We unconditionally use unqualified pointers - this is intentional as these
```

Comment:
```suggestion
  // We unconditionally use AS0 pointers - this is intentional as these
```

Since this refers to LLVM and not Clang pointer types I'd use the address space terminology.